api/testdata/complete-scenarios/test10.json

```diff
@@ -0,0 +1,5748 @@
+      "props": {
+        "Consulted": {
+          "type": "branch",
+          "props": {
+            "value": "No"
+          }
+        },
+        "List": {
+          "type": "collection",
+          "props": {
+            "branch": {
+              "type": ""
+            },
+            "items": []
+          }
+        }
+      }
+    },
+    "Diagnoses": {
+      "type": "psychological.diagnoses",
+      "props": {
+        "Diagnosed": {
+          "type": "branch",
+          "props": {
+            "value": "No"
+          }
+        },
+        "DidNotConsult": {
+          "type": "branch",
+          "props": {
+            "value": ""
+          }
+        },
+        "DiagnosisList": {
+          "type": "collection",
+          "props": {
+            "branch": {
+              "type": ""
+            },
+            "items": []
+          }
+        },
+        "InTreatment": {
+          "type": "branch",
+          "props": {
+            "value": ""
+          }
+        },
+        "TreatmentList": {
+          "type": "collection",
+          "props": {
+            "branch": {
+              "type": ""
+            },
+            "items": []
+          }
+        }
+      }
+    },
+    "ExistingConditions": {
+      "type": "psychological.conditions",
+      "props": {
+        "HasCondition": {
+          "type": "branch",
+          "props": {
+            "value": "No"
+          }
+        },
+        "ReceivedTreatment": {
+          "type": "radio",
+          "props": {
+            "value": ""
+          }
+        },
+        "Explanation": {
+          "type": "textarea",
+          "props": {
+            "value": ""
+          }
+        },
+        "TreatmentList": {
+          "type": "collection",
+          "props": {
+            "branch": {
+              "type": "branch",
+              "props": {
+                "value": ""
+              }
+            },
+            "items": []
+          }
+        },
+        "DidNotFollow": {
+          "type": "branch",
+          "props": {
+            "value": ""
+          }
+        },
+        "DidNotFollowExplanation": {
+          "type": "textarea",
+          "props": {
+            "value": ""
+          }
+        }
+      }
+    },
+    "Hospitalizations": {
+      "type": "psychological.hospitalizations",
+      "props": {
+        "Hospitalized": {
+          "type": "branch",
+          "props": {
+            "value": "No"
+          }
+        },
+        "List": {
+          "type": "collection",
+          "props": {
+            "branch": {
+              "type": ""
+            },
+            "items": []
+          }
+        }
+      }
+    }
+  },
+  "Relationships": {
+    "Cohabitants": {
+      "type": "relationships.status.cohabitant",
+      "props": {
+        "HasCohabitant": {
+          "type": "branch",
+          "props": {
+            "value": "Yes"
+          }
+        },
+        "CohabitantList": {
```

Comment:
All these cohabitants look like the are still derived from test case 5 – test case 1 is a blank canvas in this section.

Reply:
I went ahead and loaded test case 1 and then filled in the rest of the data via the ui and exported it.